services/app-web/src/pages/App/AppRoutes.tsx

```diff
@@ -69,6 +69,8 @@ const StateUserRoutes = ({
                     element={<StateDashboard />}
                 />
 
+                <Route path={RoutesRecord.HELP} element={<Help />} />
```

Comment:
Looks like we didn't have this route for CMS users as well, can you add it there? Or is there somewhere we can put it in this file that will apply everywhere rather than being repeated in all 3 log in states?

Reply:
Im looking at this yes I think we need a constant here for the routes that are user type agnostic. Also I think we should get some tests around routes.
 I'll push up something before I leave today for your review @macrael